README.md

```diff
@@ -155,9 +156,20 @@ _ <- new Jdbc4SnapshotStoreMigration(config).run()
 
 It's your choice whether you want to trigger migration manually or (recommended) leverage a database version control system of your choice (e.g. Flyway).
 
-### Examples
+#### Examples
 An example Flyway-based migration can be found in the demo app: https://github.com/mkubala/demo-akka-persistence-postgres/blob/master/src/main/scala/com/github/mkubala/FlywayMigrationExample.scala 
 
+### Migration from akka-persistence-postgres 4.0.0 to 4.1.0
```

Comment:
It's `0.4.0`, not `4.0.0`. I would like to release the version with your previous partition lookup fix as `0.4.1`.
If your changes require applying migration script, I'd release them as 0.5.0 or even 1.0.0-(RC?). Patch versions should not require any additional care, the are aimed for rather small fixes.

Reply:
Thanks, foing for 0.5.0 as I see further similar things coming soon...